pkg/kv/bulk/sst_batcher.go

```diff
@@ -664,14 +664,14 @@ func (b *SSTBatcher) Close(ctx context.Context) {
 
 // GetBatchSummary returns this batcher's total added rows/bytes/etc.
```

Comment:
nit: this docstring looks the same as `GetSummary` but returns something different.